### PR TITLE
Adding SignalR template as an item template for VS

### LINKED ISSUE #956: Add Signal R template as an item template for visual studio

### PR DESCRIPTION
**Pull request by yojagad**
Fixes #956

cc: @anthonychu (I'm not able to add you as a reviewer for some reason )

### CODE REVIEW

File: Functions.Templates/Templates/SignalRConnectionInfoHttpTrigger-CSharp/.template.config/template.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Function",
+        "Trigger",
+		    "Http",
```
Comment (soninaren) [inline Functions.Templates/Templates/SignalRConnectionInfoHttpTrigger-CSharp/.template.config/template.json:6]: Nit: Fix the formatting

File: Functions.Templates/Templates/SignalRConnectionInfoHttpTrigger-CSharp/.template.config/template.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        "SignalR"
+    ],
+    "name": "SignalRTrigger",
+    "identity": "Azure.Function.CSharp.SignalRTrigger.2.x",
+    "groupIdentity": "Azure.Function.SignalRTrigger",
```
Comment (soninaren) [inline Functions.Templates/Templates/SignalRConnectionInfoHttpTrigger-CSharp/.template.config/template.json:11]: Signal R is not a trigger, May be `Azure.Function.SignalR`

File: Functions.Templates/Templates/SignalRConnectionInfoHttpTrigger-CSharp/.template.config/template.json [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    "shortName": "signalr",
+    "tags": {
+        "language": "C#",
+        "type": "item"
+    },
+    "sourceName": "SignalRTriggerCSharp",
```
Comment (soninaren) [inline Functions.Templates/Templates/SignalRConnectionInfoHttpTrigger-CSharp/.template.config/template.json:17]: This should be SignalRConnectionInfoHttpTriggerCSharp, same as the .cs file name

File: Functions.Templates/Templates/SignalRConnectionInfoHttpTrigger-CSharp/.template.config/template.json [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+      "Description": "Adding Reference to Microsoft.Azure.WebJobs.Extensions.SignalRService NuGet package",
+      "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+      "ContinueOnError": "true",
+      "ManualInstructions": [],
+      "args": {
+        "referenceType": "package",
+        "reference": "Microsoft.Azure.WebJobs.Extensions.SignalRService", "version": "1.2.0",
+        "projectFileExtensions": ".csproj"
+      }
+    },
```
Comment (soninaren) [inline Functions.Templates/Templates/SignalRConnectionInfoHttpTrigger-CSharp/.template.config/template.json:68]: Added the same action twice

**Comment by yojagad** [comment]
Addressed code review comments.

**Pull request merged**